[2023-12-11 08:04:28,039] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_con_conexion_postgres.crear_tabla_postgres scheduled__2023-12-10T00:00:00+00:00 [queued]>
[2023-12-11 08:04:28,099] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_con_conexion_postgres.crear_tabla_postgres scheduled__2023-12-10T00:00:00+00:00 [queued]>
[2023-12-11 08:04:28,115] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-12-11 08:04:28,141] {taskinstance.py:1377} INFO - Starting attempt 2 of 6
[2023-12-11 08:04:28,365] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-12-11 08:04:28,758] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): crear_tabla_postgres> on 2023-12-10 00:00:00+00:00
[2023-12-11 08:04:28,821] {standard_task_runner.py:52} INFO - Started process 958 to run task
[2023-12-11 08:04:28,851] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_con_conexion_postgres', 'crear_tabla_postgres', 'scheduled__2023-12-10T00:00:00+00:00', '--job-id', '85', '--raw', '--subdir', 'DAGS_FOLDER/dag_postgres_database.py', '--cfg-path', '/tmp/tmp9g_7xcwl', '--error-file', '/tmp/tmpwcnwzi0t']
[2023-12-11 08:04:28,908] {standard_task_runner.py:80} INFO - Job 85: Subtask crear_tabla_postgres
[2023-12-11 08:04:29,511] {task_command.py:371} INFO - Running <TaskInstance: dag_con_conexion_postgres.crear_tabla_postgres scheduled__2023-12-10T00:00:00+00:00 [running]> on host 03477fa6e3b6
[2023-12-11 08:04:30,257] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DavidBU
AIRFLOW_CTX_DAG_ID=dag_con_conexion_postgres
AIRFLOW_CTX_TASK_ID=crear_tabla_postgres
AIRFLOW_CTX_EXECUTION_DATE=2023-12-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-12-10T00:00:00+00:00
[2023-12-11 08:04:30,350] {base.py:68} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-12-11 08:04:30,603] {dbapi.py:231} INFO - Running statement: 
            create table if not exists fin_mundo(
                dt date,
                pais varchar(30)
            )
        , parameters: None
[2023-12-11 08:04:30,702] {postgres.py:94} INFO - NOTICE:  la relación «fin_mundo» ya existe, ignorando

[2023-12-11 08:04:31,240] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=dag_con_conexion_postgres, task_id=crear_tabla_postgres, execution_date=20231210T000000, start_date=20231211T080428, end_date=20231211T080431
[2023-12-11 08:04:31,566] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-11 08:04:32,274] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
